gino/dialects/base.py

```diff
@@ -67,18 +67,58 @@ def raw_transaction(self):
 
 
 class PreparedStatement:
-    def __init__(self):
+    def __init__(self, clause=None):
         self.context = None
+        self.clause = clause
 
     def iterate(self, *params, **kwargs):
         return _PreparedIterableCursor(self, params, kwargs)
 
+    async def _do_execute(self, multiparams, params, one=False,
+                          return_model=True):
+        ctx = self.context.connection.execute(
+            self.clause, *multiparams, **params).context
+        if ctx.executemany:
+            raise ValueError('PreparedStatement does not support multiple '
+                             'parameters.')
```

Comment:
why is there such limitation?

Reply:
Because asyncpg doesn't have it on prepared statement. User who wants to use `executemany()` with prepared statement has to fall back to sequential `all()` calls, or not to use prepared statement to save RTT.

Reply:
Thx for the info. I looked up, and it seems to be due to PostgreSQL.
(I saw some "workaround" https://stackoverflow.com/a/14903130/238634 but it'd make it too complex)